src/ql/ir/old_to_new.cc

```diff
@@ -229,14 +231,16 @@ static void parse_decomposition_rule(
             "expansions"
```

Comment:
At line 63 above, there is \"breg\", it should be \"b\", since the substr(0,1) higher up results in a string of 1 long.
Can you make this patch in your branch?